.github/workflows/build-macos.yml

```diff
@@ -138,5 +105,6 @@ jobs:
         if: always()
         with:
           name: meshkernel-${{ matrix.platform }}-${{ matrix.build_type }}
-          path: ${{ steps.paths.outputs.install_dir }}
-          if-no-files-found: error
\ No newline at end of file
+          path: |
```

Comment:
No need to for a pipe here. You can write `path: ${{ steps.paths.outputs.install_dir }}`.